webrender/src/render_backend.rs

```diff
@@ -403,6 +409,15 @@ impl RenderBackend {
             }
             DocumentMsg::HitTest(pipeline_id, point, flags, tx) => {
                 profile_scope!("HitTest");
+                if doc.render_on_hittest {
```

Comment:
there are two problems with this bit:
  1. the result is not going anywhere (i.e. `Renderer` doesn't get it), which is untypical and may go across some of the implicit assumptions we have in the code
  2. nothing should be rendered inside the `DocumentMsg` arms. Instead, we return the proper `DocumentOps` struct.

Reply:
2. is incompatible with the intent of the code, which requires that the render be done prior to the hit_test. If we do the render via the DocumentOps return, that will happen after the hit test is carried out. Any suggestions on how to accomplish this?